karma.conf.js

```diff
@@ -34,15 +28,11 @@ const staticConfig = {
       base: 'Firefox',
       flags: ['--devtools']
     },
-    VirtualBoxEdgeWin10: {
-      base: 'VirtualBoxEdge',
-      keepAlive: true,
-      uuid: 'fb87b330-a1dc-4d6d-ac5b-e669912f1a4f'
-    },
     VirtualBoxIE11Win7: {
       base: 'VirtualBoxIE11',
       keepAlive: true,
-      vmName: 'IE11 - Win7'
+      vmName: 'IE11 - Win7',
+      uuid: '9c440145-fbea-49ea-a22f-4162bfaaaa2f'
```

Comment:
Yet we retain the VBox runner capabilities for IE11 due to the improved debugging capabilities in VM over grid runs